WireAPI/Package.resolved

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
question: do we need to add this one, as we always use WireAPI within monorepo Xcode workspace

Reply:
We can add it to .gitignore.
It's just automatically generated when we open the module using Xcode.

Reply:
Should I add it to .gitignore?

Reply:
no I understand let's keep it in for now